docs/openapi/components/schemas/common/AgParcelDelivery.yml

```diff
@@ -216,7 +216,71 @@ example: |-
       "phoneNumber": "555-895-1661",
       "faxNumber": "555-497-2527"
     },
-    "AgPackage": [],
+    "AgPackage": [
+      {
+        "type": [
+          "AgPackage"
+        ],
+        "packageName": "Avocados, Bulk",
+        "grade": "AA",
+        "responsibleParty": {
+          "type": "Entity",
+          "name": "Example Responsible Party Organization",
+          "email": "Chadrick_Gibson@example.com",
+          "phoneNumber": "+1-555-820-1520",
+          "entityType": "Organization"
+        },
+        "voicePickCode": "4642",
+        "date": "2021-03-14",
+        "labelImageUrl": "https://img.example.org/640/480/",
+        "labelImageHash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+        "agProduct": [
+          {
+            "type": [
+              "AgProduct"
+            ],
+            "upc": "033383401508",
+            "plu": "94225",
+            "gtin": "033383401508",
+            "product": {
+              "type": [
+                "Product"
+              ],
+              "manufacturer": {
+                "type": [
+                  "Organization"
+                ],
+                "email": "Ashlee.Grady@example.net",
+                "phoneNumber": "555-899-1399"
+              },
+              "name": "Avocados",
+              "description": "Avocados, 4 pack boxes",
+              "sizeOrAmount": {
+                "type": [
+                  "QuantitativeValue"
+                ],
+                "unitCode": "hg/ha",
+                "value": "60"
+              },
+              "weight": {
+                "type": [
+                  "QuantitativeValue"
+                ],
+                "unitCode": "hg/ha",
+                "value": "6960"
+              },
+              "sku": "81055399441"
+            },
+            "scientificName": "Persea americana",
+            "labelImageUrl": "https://img.example.org/033383401508/640/480/",
```

Comment:
again with the camel casing over object / predicate definitions... this will make queries awkward.

Reply:
Again I agree, though I think out of scope